src/k8s/requests/hosts/index.js

```diff
@@ -0,0 +1,50 @@
+import { SecretModel, BaremetalHostModel } from '../../../models';
+import { getName } from '../../../selectors';
+
+const getSecretName = name => `${name}-secret`;
+
+const getSecret = (name, namespace, username, password) => ({
+  apiVersion: 'v1',
+  kind: 'Secret',
+  metadata: {
+    namespace,
+    name,
+  },
+  data: {
+    username,
+    password,
+  },
+  type: 'Opaque',
+});
+
+const getBaremetalHostObject = (name, namespace, controller, secretName) => ({
+  apiVersion: 'metalkube.org/v1alpha1',
```

Comment:
you should get these values (apiVersion and kind) from BaremetalHostModel so when something in model is updated (version) we will have just one place to edit in web-ui and web-ui-components. Same applies for Secret above

Reply:
Done